### DISCUSSION TITLE
Di2 control

### DISCUSSION THREAD

**Discussion by Congdan** [maca88/SmartBikeLights#81] [General]
Hi,
Is it possible to cycle through light modes using di2 control?

**Comment by maca88** [comment]
Hi,
Unfortunately, it is not possible to connect di2 control with Smart Bike Lights. DI2 uses a private ANT network to communicate with a Garmin device and there is no API that could be used to intercept the button clicks. The only way to control the lights using a physical button is with a Bontrager Transmitr MicroRemote. Here is a video if interested: https://www.youtube.com/watch?v=D_vTVkWjrjI
